packages/cms-kit/src/utils/setSurrogateKeyHeader.ts

```diff
@@ -18,22 +18,30 @@ const setSurrogateKeyHeader = (
 	keys: string | null,
 	res: ServerResponse,
 ): string | void => {
+	const removeFirstTwo = (str: string) => {
+		const arr = str.split(' ');
+		const result = arr.length > 2 ? arr.slice(2) : arr;
+		return result;
+	};
+
 	if (hasHeaders(res)) {
 		const surrogateKeyHeader = res.getHeader('Surrogate-Key-Raw');
```

Comment:
I'm actually not sure if this needs to be changed to `Surrogate-Key` cc @backlineint

Reply:
Nvm, I think it makes sense to change it but I can change it back if needed.